<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Video Frame Extractor Pro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #2c3e50, stop: 1 #34495e);
    color: #ecf0f1;
}

QTabWidget::pane {
    border: 2px solid #1a252f;
    border-radius: 8px;
    background-color: #34495e;
}

QTabWidget::tab-bar {
    alignment: center;
}

QTabBar::tab {
    background-color: #16a085;
    color: white;
    padding: 12px 24px;
    margin: 4px;
    border: none;
    border-radius: 6px;
    font-weight: bold;
    font-size: 12px;
    min-width: 120px;
}

QTabBar::tab:selected {
    background-color: #1abc9c;
    color: #2c3e50;
}

QTabBar::tab:hover {
    background-color: #1abc9c;
}

QWidget {
    background-color: #34495e;
    color: #ecf0f1;
    font-family: "Segoe UI", Arial, sans-serif;
}

QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    padding: 10px 20px;
    border-radius: 6px;
    font-weight: bold;
    font-size: 12px;
    min-height: 20px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #21618c;
}

QPushButton:disabled {
    background-color: #7f8c8d;
    color: #bdc3c7;
}

QPushButton#pushButton_start {
    background-color: #27ae60;
}

QPushButton#pushButton_start:hover {
    background-color: #229954;
}

QPushButton#pushButton_stop {
    background-color: #e74c3c;
}

QPushButton#pushButton_stop:hover {
    background-color: #c0392b;
}

QLineEdit {
    background-color: #2c3e50;
    color: #ecf0f1;
    border: 2px solid #7f8c8d;
    border-radius: 6px;
    padding: 8px 12px;
    font-size: 12px;
    selection-background-color: #3498db;
}

QLineEdit:focus {
    border-color: #3498db;
}

QProgressBar {
    border: 2px solid #2c3e50;
    border-radius: 8px;
    text-align: center;
    color: #2c3e50;
    font-weight: bold;
    font-size: 12px;
}

QProgressBar::chunk {
    background-color: #1abc9c;
    border-radius: 6px;
    border: 1px solid #16a085;
}

QTextBrowser {
    background-color: #2c3e50;
    color: #ecf0f1;
    border: 2px solid #7f8c8d;
    border-radius: 8px;
    padding: 8px;
    font-family: "Consolas", "Monaco", monospace;
    font-size: 11px;
    selection-background-color: #3498db;
}

QLabel {
    color: #ecf0f1;
    font-weight: bold;
    font-size: 12px;
    padding: 4px;
}

QGroupBox {
    color: #1abc9c;
    font-weight: bold;
    font-size: 14px;
    border: 2px solid #1abc9c;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 8px;
    background-color: #34495e;
}

QFrame {
    background-color: #2c3e50;
    border-radius: 8px;
    border: 1px solid #7f8c8d;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>üé• Video ƒ∞≈üleme</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Video Ayarlarƒ±</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Hedef Video:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_input_video">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Video dosyasƒ±nƒ± se√ßin...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pushButton_select_video">
             <property name="text">
              <string>üìÅ Video Se√ß</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>√áƒ±ktƒ± Klas√∂r√º:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_output_dir">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>√áƒ±ktƒ± klas√∂r√ºn√º se√ßin...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="pushButton_select_output_dir">
             <property name="text">
              <string>üìÇ Klas√∂r Se√ß</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>ƒ∞≈ülem Kontrolleri</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_start">
             <property name="text">
              <string>üöÄ Ayrƒ±≈ütƒ±rmayƒ± Ba≈ülat</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_stop">
             <property name="text">
              <string>‚èπÔ∏è Ayrƒ±≈ütƒ±rmayƒ± Durdur</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>ƒ∞lerleme Durumu</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_progress">
             <property name="text">
              <string>Hazƒ±r...</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar_status_bar">
             <property name="value">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="format">
              <string>%p%</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>ƒ∞≈ülem Loglarƒ±</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextBrowser" name="textBrowser_output">
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas','Monaco'; font-size:11px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>‚ÑπÔ∏è Hakkƒ±nda</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_about_title">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;center&gt;
&lt;h1 style=&quot;color:#1abc9c;&quot;&gt;Video Frame Extractor Pro&lt;/h1&gt;
&lt;h3 style=&quot;color:#3498db;&quot;&gt;Profesyonel Video Kare Ayrƒ±≈ütƒ±rma Yazƒ±lƒ±mƒ±&lt;/h3&gt;
&lt;/center&gt;
&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>üìã Uygulama Bilgileri</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Geli≈ütirici:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_developer">
             <property name="text">
              <string>Mehmet Y√ºksel ≈ûekeroƒülu</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #1abc9c;
font-weight: bold;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Versiyon:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_version">
             <property name="text">
              <string>v1.0.0</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #3498db;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Derleme Tarihi:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_build_date">
             <property name="text">
              <string>2025</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Qt Versiyonu:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_qt_version">
             <property name="text">
              <string>6.5.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>üìú Lisans Bilgisi</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_license">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;center&gt;
&lt;h3 style=&quot;color:#e74c3c;&quot;&gt;MIT License&lt;/h3&gt;
&lt;/center&gt;
&lt;p&gt;Copyright ¬© 2025 Mehmet Y√ºksel ≈ûekeroƒülu&lt;/p&gt;
&lt;p&gt;Bu lisans, kullanƒ±cƒ±lara a≈üaƒüƒ±daki haklarƒ± vermektedir:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;√úcretsiz kullanƒ±m&lt;/li&gt;
&lt;li&gt;Deƒüi≈ütirme ve modifikasyon&lt;/li&gt;
&lt;li&gt;Daƒüƒ±tƒ±m&lt;/li&gt;
&lt;li&gt;Ticari kullanƒ±m&lt;/li&gt;
&lt;/ul&gt;
&lt;p&gt;&lt;strong&gt;Ko≈üullar:&lt;/strong&gt; Lisans metni ve telif hakkƒ± bildirimi t√ºm kopyalarda bulunmalƒ±dƒ±r.&lt;/p&gt;
&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignTop</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>üõ†Ô∏è Teknolojiler</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;ul&gt;
&lt;li&gt;Qt Framework&lt;/li&gt;
&lt;li&gt;OpenCV&lt;/li&gt;
&lt;li&gt;Multithreading&lt;/li&gt;
&lt;li&gt;Modern C++&lt;/li&gt;
&lt;/ul&gt;
&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;ul&gt;
&lt;li&gt;Real-time Processing&lt;/li&gt;
&lt;li&gt;High Performance&lt;/li&gt;
&lt;li&gt;User Friendly UI&lt;/li&gt;
&lt;li&gt;Cross Platform&lt;/li&gt;
&lt;/ul&gt;
&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>üìÑ Dosya</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuYard_m">
    <property name="title">
     <string>‚ùì Yardƒ±m</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menuYard_m"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: #2c3e50;
color: #ecf0f1;
font-weight: bold;</string>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>üö™ √áƒ±kƒ±≈ü</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>‚ÑπÔ∏è Hakkƒ±nda</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>